module/x/gravity/keeper/querier.go

```
@@ -31,7 +31,7 @@ func NewQuerier(keeper Keeper) sdk.Querier {
 }
 
 func queryCurrentValset(ctx sdk.Context, keeper Keeper) ([]byte, error) {
-	valset := keeper.GetCurrentValset(ctx)
+	valset, _ := keeper.GetCurrentValset(ctx)
```

this should return an error if it receives one.